firefox-ios/Client/Frontend/Browser/Toolbars/ToolbarFlagManager.swift

```diff
@@ -10,4 +10,9 @@ struct ToolbarFlagManager {
         return LegacyFeatureFlagsManager.shared.isFeatureEnabled(.toolbarRefactor,
                                                                  checking: .buildOnly)
     }
+
+    static var isOneTapNewTabEnabled: Bool {
+        return LegacyFeatureFlagsManager.shared.isFeatureEnabled(.toolbarOneTapNewTab,
+                                                                 checking: .buildOnly)
+    }
```

Comment:
nit: curious, why not use `FeatureFlaggable` instead of using the shared instance directly?

Reply:
I just looked at how things were done for TabTray. I removed the `ToolbarFlagManager` now and use `FeatureFlaggable` instead.